src/compiler/python_generator.cc

```diff
@@ -101,6 +101,11 @@ PrivateGenerator::PrivateGenerator(const GeneratorConfiguration& config,
 void PrivateGenerator::PrintAllComments(StringVector comments,
                                         grpc_generator::Printer* out) {
   if (comments.empty()) {
+    // It's important we print something here and not omit this line,
```

Comment:
These four lines could use some wordsmithing. Is it guaranteed at this point in the code that the service has no methods and thus a failure to print the pass would create a syntactically invalid module?